app/ai/voice/tts/dragontts.py

```
@@ -0,0 +1,277 @@
+                        num_channels=1,
+                        context_id=context_id,
+                    )
+            # A trailing odd byte (carry is always 0 or 1 byte here) can't form a
+            # complete 16-bit sample, so drop it rather than emit a corrupt frame.
+        except httpx.HTTPStatusError as e:
+            detail = e.response.text[:200]
+            logger.error(f"DragonTTS returned HTTP {e.response.status_code}: {detail}")
+            yield ErrorFrame(
+                error=f"DragonTTS returned HTTP {e.response.status_code}: {detail}"
+            )
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)